src/Jupyter/Visualization/DisplayableExceptionEncoders.cs

```diff
@@ -74,10 +90,12 @@ public class DisplayableExceptionToHtmlEncoder : IResultEncoder
                         {String.Join("\n", rows)}
                     </tbody>
                 ";
+
+                // { display: list-item } style is needed for a <summary> tag so that the arrow shows up 
```

Comment:
Jupyter's stylesheets (I think) were causing the disclosure triangle to be hidden, so it wasn't clear you could click on the element to see the callstack.

With this change, it should look like:

![image](https://user-images.githubusercontent.com/16928427/221940131-4359c360-c256-4a90-98f0-d707262058ad.png)

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary#default_style